main.go

```diff
@@ -91,12 +94,86 @@ func getAllRecordsHandler(w http.ResponseWriter, r *http.Request, pr httprouter.
     }
 }
 
+func createPlayer(w http.ResponseWriter, r *http.Request, pr httprouter.Params) {
```

Comment:
使っていない引数は，`_`と明示的に書いておく方が良いようです
```suggestion
func createPlayer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```